db/schema.rb

```diff
@@ -22,8 +22,11 @@
     t.string   "name"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "active_turn"
   end
 
+  add_index "games", ["active_turn"], name: "index_games_on_active_turn", using: :btree
```

Comment:
We don't need the brackets for `["active_turn"]`. They are needed when creating an index on more than 1 column. 

Also, any reason to specify `using: :btree`? I looked it up and it seems it uses btree by default for Postgres. 

Also, come to think of it, I don't think we need an index an active_turn since we will be looking up active_turn by game_id and not the other way around.